python-sdk/src/langwatch/client.py

```
@@ -751,6 +763,51 @@ def _setup_rest_api_client(self) -> LangWatchApiClient:
+        if ssl_verify in ("false", "0", "no", "disable"):
+            if not Client._ssl_config_warned:
+                Client._ssl_config_warned = True
+                logger.info(
+                    "LANGWATCH_SSL_VERIFY is disabled. SSL certificate "
+                    "verification will be skipped."
+                )
+            return False  # type: ignore[return-value]  # OTEL exporter passes this as verify= to requests
+
+        if ca_bundle:
+            if os.path.isfile(ca_bundle):
+                logger.info(f"Using custom CA bundle: {ca_bundle}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)